packages/msw-addon/src/preset/addParameter.js

```diff
@@ -0,0 +1,5 @@
+export const parameters = {
+  msw: {
+    handlers: []
```

Comment:
This is using the possible new notation for handlers, so this PR would require the other one to be merged first